src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -138,6 +138,21 @@ class TransactionMRI(Enum):
     SET_ON_DEMAND = "s:transactions/on_demand@none"
 
 
+class SpanMRI(Enum):
+    USER = "s:spans/user@none"
```

Comment:
I left these in the `TransactionMRI` as well, just to keep the blast radius of this PR smaller. We can switch the other references at a later date.